utils/scripts/helpers/build_url_repo.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
These new scripts (and possibly the whole of utils/scripts/helpers or even utils/scripts) should be added to the paths entry linked below so changes to them trigger more comprehensive testing in GitHub Actions.

https://github.com/daos-stack/daos/blob/master/.github/workflows/landing-builds.yml#L10

Reply:
Done